### PR TITLE
Render MTA-STS adoption charts

### PR DESCRIPTION
**Pull request by vbrown608**
@sydneyli feel free to push wording changes!

(Actually) marking this WIP because it depends on https://github.com/EFForg/starttls-backend/pull/232 and co.

### CODE REVIEW

File: content/about.md [hunk 1]
```diff
@@ -24,4 +24,13 @@ In 2015, researchers discovered that ~20% of the Alexa top million domains which
 
 These results are the motivation for this project.
 
-<small><sup>[1]</sup> Neither Snow Nor Rain Nor MITM . . . An Empirical Analysis of Email Delivery Security https://zakird.com/papers/mail.pdf</small>
+<strong class="js-yes">Percent of scans on starttls-everywhere.org supporting MTA-STS</strong>
+<canvas id="visitors-chart" height="100"></canvas>
+
+<strong class="js-yes">Domains in the top million<sup>[2]</sup> supporting MTA-STS</strong>
+<canvas id="top-million-chart" height="100"></canvas>
+
```
Comment (sydneyli) [inline content/about.md]: Let's move these charts up above the `Statistics` section and add the following text before the charts:

```html
<h5>MTA-STS adoption and integration</h5>
MTA-STS is a new Internet standard for e-mail that has many of the same goals as STARTTLS Everywhere. We're tracking its adoption across the top million domains and users of our site. Here are some growth statistics about the deployment of MTA-STS!
```

I also realized I never pushed the new drafts of MTA-STS FAQ, etc. Let me see if I can find the branch!

File: js/stats.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+  $.ajax({
+    type: 'GET',
+    url: hostname + '/api/stats',
+  })
+  .done(function(resp) {
+    load_chart('top-million-chart', resp.response.TOP_DOMAINS, '#f938bd', function(item) {
+      return item.value + ' domains';
+    });
+    load_chart('visitors-chart', resp.response.LOCAL, '#20c9c9', function(item) {
```
Comment (sydneyli) [inline js/stats.js]: Is there a way to constant-ize these hex colors?

Comment (vbrown608) [reply js/stats.js]: There's not a way I'm super into to share them with the CSS but I can put them in js vars further up!

Comment (lschatzkin) [reply js/stats.js]: It'd be cool if we could use the same variable names that are in variable.scss.

**Comment by klausenbusk** [comment]
Was this ever "deployed"? https://starttls-everywhere.org/about/ does not show the charts. The new PGP instructions isn't shown either (https://github.com/EFForg/starttls-frontend/pull/284).

**Comment by sydneyli** [comment]
Huh! Something must be up. I'll take a look at our deploy process. Thanks for flagging @klausenbusk !

**Pull request merged**